dashboard/modules/metrics/dashboards/serve_deployment_dashboard_panels.py

```diff
@@ -28,7 +28,7 @@
         unit="qps",
         targets=[
             Target(
-                expr='sum(rate(ray_serve_deployment_request_counter{{route=~"$Route",route!~"/-/.*",{global_filters}}}[5m])) by (deployment, replica)',
+                expr='sum(rate(ray_serve_deployment_request_counter{{route=~"$Route",route!~"/-/.*",{global_filters}}}[5m])) by (application, deployment, replica)',
                 legend="{{replica}}",
```

Comment:
does the legend need to change to include application name? Replica names include deployment name and replica id, seems application name is missing.

Reply:
With the changes from https://github.com/ray-project/ray/pull/38536, replica names will now include application name!